packages/shared/scope-hoisting/src/link.js

```diff
@@ -602,6 +614,7 @@ export function link({
           replacements,
           options,
         );
+        path.scope.crawl();
```

Comment:
I assume this is because `generateExports` might mess with the scope?

Reply:
It did, but in this case I just forgot to remove the call after making `generateExports` maintain the scope